docs/docs_style_guide.md

```diff
@@ -0,0 +1,479 @@
+
+## O
+
+### online
+
+One word.
+
+### online services
+
+Lower case unless part of a proper noun.
+
+## P
+
+### Per cent
+
+Use per cent not percent. Percentage is 1 word. Always use % with a
+number.
+
+### plain English
+
+All content in LiveCode documentation should be written in plain
+English. You should also make sure you use language your audience
+will understand.
+
+## Q
+
+### Quotes and speech marks
+
+In long passages of speech, open quotes for every new paragraph, but
+close quotes only at the end of the final paragraph.
+
+#### Single quotes
+
+Use single quotes:
+
+* in headlines
+* for unusual terms
+* when referring to words or publications, for example: 'Download the
+  'LiveCode Builder Reference Manual' (PDF, 360KB)'
+
+#### Double quotes
+
+Use double quotes in body text for direct quotations.
+
+#### Block quotes
+
+Use the block quote Markdown for quotes longer than a few sentences.
+
+## R
+
+### References
+
+References should be easy to understand by anyone, not just
+specialists.
+
+They should follow the style guide. When writing a reference:
+
+* don't use italics
+* use single quote marks around titles
+* write out abbreviations in full, for example 'page' not 'p', and
+  'IEEE Transactions on Computers' not 'IEEE Trans. Comp.'
+* use plain English, for example use 'and others' not 'et al'
+* don't use full stops after initials or at the end of the reference
+
+If the reference is available online, make the title a link and
+include the date you accessed the online version.
+
+## S
+
+### Scrum
+
+Upper case when referring to the framework and methon for developing
+products, otherwise use lower case.
+
+### seasons
+
+spring, summer, autumn and winter are lowercase.
+
+### Sentence length
+
+Don't use long sentences. Check and sentences with more than 25 words
+to see if you can split them to make them clearer.
+
+### spaces
+
+Use only one space after a full stop, not 2.
+
+### Speech marks
+
+See 'Quotes and speech marks'.
+
+### steps
+
+See 'Bullets and steps'.
+
+### Summaries
+
+Summaries should:
+
+* be 140 characters or less
+* end with a full stop
+* not repeat the title or body text
+* be clear and specific
+
+## T
+
+### technical terms
+
+Where you need to use technical terms, you can. They're not
+jargon. You just need to explain what they mean the first time you use
+them.
+
+### Temperature
+
+Use Celcius, e.g. 37℃.
+
+### Times
+
+* use 'to' in time ranges, not hyphens, en rules or em dashes. For example, '10am to 11am' not '10–11am'.
+* 5:30pm (not 1730hrs)
+* midnight, not 00:00
+* midday, not 12 noon, noon or 12pm
+* 6 hours 30 minutes
+
+### Titles
+
+Titles should:
+
+* be 65 characters or less
+* be unique, clear and descriptive
+* be front-loaded and optimised for search
+* use a colon to break up longer titles
+* not contain dashes or slashes
+* not have a full stop at the end
+* not use acronyms unless they are well-known, eg EU
+
+## W
+
+### Webpage
+
+One word.
```

Comment:
Q for consistency: why is "homepage" necessarily lowercase while "Webpage" is not?